src/compas_rhino/interop/__init__.py

```diff
@@ -0,0 +1,175 @@
+"""
+********************************************************************************
+interop
```

Comment:
`interop` doesn't sound right, the whole of compas is interoperable with rhino, this package is more specific, is really about conversions, I would be explicit in its name as well, e.g. `compas_rhino.conversions` or `compas_rhino.convert` .